code/script.js

```diff
@@ -1,9 +1,75 @@
 // A function that adds and remove the class "active" on the section you click on.
 // We haven't really talked about ´this´ yet, but we will... ;)
 // console.log(this) to see how it works
+
+
+
+
+
 function toggle() {
   this.classList.toggle("active")
 }
 
-// Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+$('.form').find('input, textarea').on('keyup blur focus', function (e) {
```

Comment:
Seems like that you are using a framework called jQuery here! That's great if you want, but you can do the same with vanilla javascript as well if you want to practice it!